rkt/fetch_keys.go

```diff
@@ -127,6 +127,9 @@ func downloadKey(url string) (*os.File, error) {
 // addKeys adds the keys listed in pkls at prefix
 func addKeys(pkls []string, prefix string, allowHTTP, forceAccept, allowOverride bool) error {
 	ks := getKeystore()
+	if ks == nil {
+		return fmt.Errorf("could not get the key store")
```

Comment:
Exact... addKeys() is called by either `rkt trust` or `rkt fetch/prepare/run` and they both check `globalFlags.InsecureSkipVerify`.

Should I turn this into a panic() or is it fine like this?